docs/contracts/abstract-contracts.rst

```diff
@@ -6,27 +6,27 @@
 Abstract Contracts
 ******************
 
-Contracts are marked as abstract when at least one of their functions lacks an implementation as in the following example (note that the function declaration header is terminated by ``;``)::
+Contracts need to be marked as abstract when at least one of their functions lacks an implementation as in the following example (note that the function declaration header is terminated by ``;``)::
```

Comment:
What about the converse? Can you mark a contract as abstract when all functions have an implementation? Please also explain what "mark as abstract" means specifically.